lib/elixir/lib/kernel.ex

```diff
@@ -1347,6 +1347,8 @@ defmodule Kernel do
               :erlang.error atom.exception([])
             %{__struct__: struct, __exception__: true} = other when is_atom(struct) ->
               :erlang.error other
+            _ ->
+              :erlang.error ArgumentError.exception("raise/1 only accepts module names or binaries")
```

Comment:
I think the "common" wording would be "raise/1 expects an alias or a string as first argument"